src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -106,6 +98,27 @@ public async Task StartContainerAsync(IExecutionContext executionContext, object
                 throw new NotSupportedException(StringUtil.Loc("MinRequiredDockerClientVersion", requiredDockerVersion, _dockerManger.DockerPath, dockerVersion.ClientVersion));
             }
 
+            executionContext.Debug($"Delete stale container networks from previous jobs");
```

Comment:
any reason run networkprune before delete container?
i think in order to make network prune work, all referenced containers needs to gone first.

Reply:
Fixed